PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -242,60 +241,128 @@ for STRING in "$AtlasSpaceFolder"/"$NativeFolder"@native@roi "$AtlasSpaceFolder"
   done
 done
 
-BiasFieldComputed=false
-#Reduce memory usage by smoothing on downsampled mesh (match the gifti version by using the first lowresmesh)
-LowResMesh="${LowResMeshesArray[0]}"
-# myelin map only loop
-for MyelinMap in MyelinMap SmoothedMyelinMap ; do
-	if [ "$BiasFieldComputed" = false ]; then
-		# ----- Begin moved statements -----
-		# Recompute Myelin Map Bias Field Based on Better Registration
-		log_Msg "Recompute Myelin Map Bias Field Based on Better Registration"
+# BC processing
+if [ "${T2wPresent}" = "YES" ] ; then	
+	# Create surface on HighResMesh in subject's T1w space
+	${CARET7DIR}/wb_command -surface-resample ${StudyFolder}/${Subject}/T1w/${NativeFolder}/${Subject}.L.midthickness.native.surf.gii \
+		${AtlasSpaceFolder}/${NativeFolder}/${Subject}.L.sphere.MSMSulc.native.surf.gii \
+		${AtlasSpaceFolder}/${Subject}.L.sphere.${HighResMesh}k_fs_LR.surf.gii \
+		BARYCENTRIC ${StudyFolder}/${Subject}/T1w/${Subject}.L.midthickness.${HighResMesh}k_fs_LR.surf.gii
```

Comment:
If we aren't going to delete these high res T1w surfaces, we should probably generate them unconditionally, rather than only when T2w images are present.  Don't create differences between datasets when you don't have to.